src/test/java/cosminsechel/tests/tema8/testpage/TestPages.java

```diff
@@ -0,0 +1,57 @@
+package cosminsechel.tests.tema8.testpage;
+
+import org.testng.annotations.Test;
+import cosminsechel.tests.tema8.pages.*;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotEquals;
+
+public class TestPages extends BaseTest {
+
+    @Test(groups = {"desktop", "mobile"})
+    public void testBasicAjaxPage() {
+        BasicAjaxPage basicAjaxPage = new BasicAjaxPage(driver);
+        basicAjaxPage.goToPage();
+        basicAjaxPage.selectOption("Server");
+        basicAjaxPage.selectLanguage("Java");
+        basicAjaxPage.clickCodeItInButton();
+        assertEquals(basicAjaxPage.getSelectedOption(),"23");
+    }
+
+    @Test(groups = {"desktop", "mobile"})
+    public void javascriptRedirectPage() {
+        JavaScriptRedirectPage javascriptRedirectPage = new JavaScriptRedirectPage(driver);
+        javascriptRedirectPage.goToPage();
+        javascriptRedirectPage.clickRedirectButton();
+        javascriptRedirectPage.waitForRedirect();
+        assertEquals("Redirected Landing Page", driver.getTitle());
+        driver.navigate().back();
```

Comment:
No need to navigate back. You can remove this line or add an assert to check after it to check that the url has changed. Thanks!